tests/Tags/Concerns/RendersFormsTest.php

```diff
@@ -89,6 +93,306 @@ public function it_minifies_space_between_field_html_elements()
 
         $this->assertEquals($expected, $this->tag->minifyFieldHtml($fields));
     }
+
+    private function createField($type, $value, $default, $old, $config = [])
+    {
+        $config = array_merge($config, ['type' => $type]);
+
+        if ($default) {
+            $config['default'] = $default;
+        }
+
+        $field = new Field('test', $config);
+        $field->setValue($value);
+
+        if ($old !== self::MISSING) {
+            session()->flashInput(['test' => $old]);
+            $this->get('/'); // create a request so the session works.
+        }
```

Comment:
Can't just pass `null` because of the explanation in the other comment. We need to be able to tell the difference between `null` and completely missing.